setup.py

```diff
@@ -56,13 +56,21 @@
         'Operating System :: POSIX :: Linux',
     ],
 
-    packages=['bigchaindb', 'bigchaindb.commands', 'bigchaindb.db'],
+    packages=[
+        'bigchaindb',
+        'bigchaindb.commands',
+        'bigchaindb.db',
+        'bigchaindb.consensus'
+    ],
 
     entry_points={
         'console_scripts': [
             'bigchaindb=bigchaindb.commands.bigchain:main',
             'bigchaindb-benchmark=bigchaindb.commands.bigchain_benchmark:main'
         ],
+        'bigchaindb.plugins': [
+            'base=bigchaindb.consensus.base:ConsensusRules'
```

Comment:
`consensus` might be a better name than `base`, it'd be easier to understand IMO. Ideas welcome.